src/osd/OSD.cc

```diff
@@ -1825,6 +1827,11 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
       ss << "op_tracker tracking is not enabled now, so no ops are tracked currently, even those get stuck. \
 	Please enable \"osd_enable_op_tracker\", and the tracker will start to track new ops received afterwards.";
     }
+  } else if (command == "dump_historic_long_ops") {
```

Comment:
this command also needs to be registered and unregsitered.. search for 'dump_history_ops' in this file to see where/how that happens